src/main/java/com/aws/greengrass/easysetup/DeviceProvisioningHelper.java

```diff
@@ -116,6 +116,7 @@ public class DeviceProvisioningHelper {
                     + "    ]\n"
                     + "}";
     private static final String ROOT_CA_URL = "https://www.amazontrust.com/repository/AmazonRootCA1.pem";
+    private static final String CA_3_URL = "https://www.amazontrust.com/repository/AmazonRootCA3.pem";
```

Comment:
nit: `ROOT_CA_3_URL`
lets keep the naming consistent

Reply:
Updated